tester/src/test/scala/spinal/tester/scalatest/FormalArbiterTester.scala

```diff
@@ -0,0 +1,455 @@
+package spinal.tester.scalatest
+
+import spinal.core._
+import spinal.core.formal._
+import spinal.lib._
+import spinal.lib.formal._
+
+class FormalArbiterTester extends SpinalFormalFunSuite {
+  test("Arbiter-sequentialOrder-verify") {
+    FormalConfig
+      .withBMC(20)
+      .withProve(20)
+      .withCover(20)
+      // .withDebug
+      .doVerify(new Component {
+        val portCount = 5
+        val dataType = Bits(8 bits)
+
+        val dut = FormalDut(
+          new StreamArbiter(dataType, portCount)(StreamArbiter.Arbitration.sequentialOrder, StreamArbiter.Lock.none)
+        )
+        val reset = ClockDomain.current.isResetActive
+
+        assumeInitial(reset)
+
+        val select = UInt(log2Up(portCount) bit)
+        val selectOH = Bits(portCount bit)
+        val inputs = Vec(slave(Stream(dataType)), portCount)
+        val output = master(Stream(dataType))
+
+        when(reset || past(reset)) {
+          for (i <- 0 until portCount) {
+            assume(inputs(i).valid === False)
+          }
+        }
+
+        select := dut.io.chosen
+        selectOH := dut.io.chosenOH
+        output << dut.io.output
+
+        for (i <- 0 until portCount) {
+          inputs(i) >> dut.io.inputs(i)
+          inputs(i).formalAssumesSlave()
+        }
+        output.formalCovers(2)
+        output.formalAssertsMaster()
+
+        assert(select < portCount)
+        assert(select === OHToUInt(selectOH))
+        assert(output.payload === inputs(select).payload)
```

Comment:
For arbiter, it's better to ensure the unselected stream is not valid. Also, only the selected stream's ready signal would pass.